src/ui/public/agg_response/tabify/tabify.js

```diff
@@ -43,31 +43,31 @@ export function tabifyAggResponse(aggs, esResponse, respOpts = {}) {
  * @returns {undefined}
  */
 function collectBucket(write, bucket, key, aggScale) {
-  const agg = write.aggStack.shift();
+  const column = write.aggStack.shift();
+  const agg = column.aggConfig;
   const aggInfo = agg.write(write.aggs);
   aggScale *= aggInfo.metricScale || 1;
 
   switch (agg.type.type) {
     case 'buckets':
       const buckets = new TabifyBuckets(bucket[agg.id], agg.params);
       if (buckets.length) {
-        const splitting = write.canSplit && agg.schema.name === 'split';
-        if (splitting) {
-          write.split(agg, buckets, function forEachBucket(subBucket, key) {
-            collectBucket(write, subBucket, agg.getKey(subBucket, key), aggScale);
-          });
-        } else {
-          buckets.forEach(function (subBucket, key) {
-            write.cell(agg, agg.getKey(subBucket, key), function () {
-              collectBucket(write, subBucket, agg.getKey(subBucket, key), aggScale);
-            }, subBucket.filters);
-          });
-        }
+        buckets.forEach(function (subBucket, key) {
+          const bucketValue = agg.getKey(subBucket, key);
+          const hasBucketValue = typeof bucketValue !== 'undefined';
+          if (hasBucketValue) {
+            write.bucketBuffer.push({ id: column.id, value: bucketValue });
+          }
+          collectBucket(write, subBucket, agg.getKey(subBucket, key), aggScale);
+          if (hasBucketValue) {
+            write.bucketBuffer.pop();
+          }
+        });
       } else if (write.partialRows && write.metricsForAllBuckets && write.minimalColumns) {
```

Comment:
Are ~`partialRows`~ (in the Inspector) and `metricsForAllBuckets` still used anywhere? Looking through the code it seems, both are now not used anymore and we can remove them.

Reply:
yeah, this is an assumption i made (that this is not used anywhere), forgot to remove it from here. so from now on we only have minimalColumns setting, which is true by default.

setting it to false will produce table with all the metrics after every bucket 
`geo.src   response.raw  avg-bytes  count` would be
`geo.src  avg-bytes  count  response.raw  avg-bytes  count`